cmake/modules/FindPySpark.cmake

```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+#
+# Try to find pyspark
+# Once done this will define
+#
+#  PYSPARK_FOUND - system has pyspark and its dependencies are met
+#
+
+message(STATUS "Looking for pyspark dependency: Java")
+find_package(Java 1.8 COMPONENTS Runtime)
```

Comment:
Would `import pyspark` work without java runtime installed? If yes: I agree, we need to check for the Java runtime. If it would fail, let's just rely on `pyspark` checking it?

Reply:
Unfortunately `import pyspark` doesn't error out if Java runtime is not installed on the system. It only errors out during runtime of a pyspark application